packages/terra-functional-testing/src/config/wdio.conf.js

```diff
@@ -157,12 +159,43 @@ exports.config = {
   // Test reporter for stdout.
   // The only one supported by default is 'dot'
   // see also: https://webdriver.io/docs/dot-reporter.html
-  reporters: ['spec', AccessibilityReporter],
+  reporters: ['spec', AccessibilityReporter, SpecReporter],
   //
   // Options to be passed to Mocha.
   // See the full list at http://mochajs.org/
   mochaOpts: {
     ui: 'bdd',
     timeout: 60000,
   },
+  /**
+   * Gets executed once before all workers get launched.
+   * @param {Object} config - wdio configuration object.
+   * @param {Array.<Object>} capabilities - List of capabilities details.
+   */
+  onPrepare(config) {
+    const { reporterOptions = {} } = config;
+    const { outputDir } = reporterOptions;
+
+    // Remove previous reporter results.
+    cleanResults(outputDir);
```

Comment:
Can this be moved into the reporter's initalization?

Reply:
No, a reporter is initialized for each spec file so it would be invoked multiple times (once for each spec file in a project). It needs to happen once before the worker processes start. This utility removes all the previous spec result files.

Reply:
This would add complexity, but could you use a singleton in the reporter to cleanResults for a dir? It might be cleaner from a config perspective, but might run into concurrency issues.

Reply:
I spiked into the singleton idea a little more, I think for now my vote is not to implement the singleton. I believe there could be some concurrency issues that would be challenging to debug and ensure consistency.